e2e-tests/tests/async-ignore-annotations/02-assert.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 120
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    async-ignore-labels-mysql: "true"
+    async-not-ignore-labels-mysql: "true"
+    app.kubernetes.io/component: mysql
+    app.kubernetes.io/instance: async-ignore-annotations
+    app.kubernetes.io/managed-by: percona-server-operator
+    app.kubernetes.io/name: percona-server
+    app.kubernetes.io/part-of: percona-server
+    percona.com/exposed: "true"
+  annotations:
+    cloud.google.com/neg: '{"ingress":true}'
+    async-ignore-annotations-mysql: "true"
```

Comment:
Shouldn't this fail if we didn't yet set what we want to ignore? If I understand correctly if you don't set what you want to ignore operator removes it on the next reconcile loop.

Reply:
No) It should work like this now: 
If you don't have any serviceAnnotations/Labels operator doesn't track custom annotations/labels and as the result they stay in service. When you add Annotations/Labels to service operator starts to track them and keep only annotations from service (added via CR) or custom annotations which we added in ignore.

Reply:
Ok